anchor-reference-server/src/main/java/org/stellar/anchor/reference/event/SqsListener.java

```diff
@@ -0,0 +1,108 @@
+package org.stellar.anchor.reference.event;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import com.amazonaws.services.sqs.model.*;
+import com.google.gson.Gson;
+import java.util.List;
+import java.util.concurrent.*;
+import javax.annotation.PreDestroy;
+import org.stellar.anchor.event.models.QuoteEvent;
+import org.stellar.anchor.event.models.TransactionEvent;
+import org.stellar.anchor.reference.config.SqsListenerSettings;
+import org.stellar.anchor.util.Log;
+
+public class SqsListener extends AbstractEventListener {
+  private final AnchorEventProcessor processor;
+  private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(5);
+  private final AmazonSQSAsync sqsClient;
+
+  public SqsListener(SqsListenerSettings sqsListenerSettings, AnchorEventProcessor processor) {
+    this.processor = processor;
+    this.sqsClient =
+        AmazonSQSAsyncClientBuilder.standard()
+            .withRegion(sqsListenerSettings.getRegion())
+            .withCredentials(
+                new AWSStaticCredentialsProvider(
+                    new BasicAWSCredentials(
+                        sqsListenerSettings.getAccessKey(), sqsListenerSettings.getSecretKey())))
```

Comment:
There are multiple ways that AWS credentials can be configured. We probably need to re-factor all the secret configurations after this PR is merged.